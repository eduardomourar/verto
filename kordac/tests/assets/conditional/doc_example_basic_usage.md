{conditional if condition="version >= 0.2.0"}

This content only appears if version is greater than 0.2.

{conditional end}
