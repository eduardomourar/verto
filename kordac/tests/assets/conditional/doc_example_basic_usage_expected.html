{% if version >= 0.2.0 %}
<p>
 This content only appears if version is greater than 0.2.
</p>
{% endif %}
